.eslintrc

```diff
@@ -0,0 +1,52 @@
+{
+  "env": {
+    "amd": true,
+    "es6": true,
+    "jasmine": true,
+    "node": true,
+    "mocha": true,
+    "browser": true,
+    "builtin": true
+  },
+  "rules": {
+    "camelcase": 0,
+    "comma-dangle": 0,
+    "complexity": [2, 6],
+    "consistent-return": 0,
+    "curly": [2, "all"],
+    "dot-notation": 0,
+    "eqeqeq": [2, "allow-null"],
+    "global-strict": [0, "never"],
+    "globals": {
+      "define": true,
```

Comment:
IIRC, we don't need to add `define` as a global here if we have the "amd" `env`.